test/test_linalg.py

```diff
@@ -96,7 +97,31 @@ def get_tunableop_validators():
         validators[key] = value
     return validators
 
+def find_tunableop_result(results, OpSig, ParamSig):
+    assert isinstance(results, tuple)
+    for inner_tuple in results:
+        if OpSig in inner_tuple:
+            if ParamSig in inner_tuple:
```

Comment:
lint didn't say anything about this?  does this fit on one line?
```suggestion
        if OpSig in inner_tuple and ParamSig in inner_tuple:
```